src/lib/Hydra/Controller/JobsetEval.pm

```diff
@@ -119,6 +120,9 @@ sub view_GET {
         } else {
             push @{$c->stash->{new}}, $build if !$found;
         }
+        if ($build->buildstatus != 0) {
+            push @{$c->stash->{failed}}, $build;
+        }
```

Comment:
Since in the end we just count how many we have, we could make `failed = 0` above and increment it here.

We may also want to extend this check: if buildstatus is defined AND non-zero. It is extremely edge-case-ey but consider a scenario where builds are queued and none are failed: I think `failed` would be >0 in that scenario.